mautic/mautic
# Fix flip array for groups in select in segments filter

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging for features or enhancements / 3.0 for bug fixes 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Issue coming from our plugin what add new segment filters. Probably code review should enough and  check If this change not ruined any select/multiselect/country field in segment filter




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9339"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
